packages/retail-ui/components/Modal/Modal.tsx

```diff
@@ -198,7 +198,9 @@ export default class Modal extends React.Component<ModalProps, ModalState> {
                 <ResizeDetector onResize={this.handleResize}>
                   <FocusLock disabled={this.isDisableFocusLock()} autoFocus={false}>
                     {!hasHeader && !this.props.noClose ? (
-                      <Close requestClose={this.requestClose} disableClose={this.props.disableClose} />
+                      <ZIndex priority={'ModalCross'} className={cx(jsStyles.closeWrapper())}>
```

Comment:
для одного класса функция `cx()` не нужна.
```suggestion
                      <ZIndex priority={'ModalCross'} className={jsStyles.closeWrapper()}>
```